tempesta_fw/t/functional/very_many_backends/test_deadtime_1M.py

```diff
@@ -0,0 +1,255 @@
+            listener.backlog = 9000
+        config.build_config()
+        return
+
+    def create_servers(self):
+        self.servers = []
+        #default server
+        defport=tempesta.upstream_port_start_from()
+        server = control.Nginx(listen_port=defport)
+        self.setup_nginx_config(server.config)
+        self.servers.append(server)
+
+        server = control.Nginx(listen_port=self.base_port,
+                               listen_ports=self.num_attempts,
+                               listen_ip=self.ips[0])
+        self.setup_nginx_config(server.config)
+        self.servers.append(server)
+
+        self.extra_servers_base = len(self.servers)
+        for ifc in range(self.num_extra_interfaces):
+            server = control.Nginx(listen_port=self.base_port,
+                                   listen_ports=self.num_extra_ports,
+                                   listen_ip=self.ips[ifc + 1])
+            self.setup_nginx_config(server.config)
+            self.servers.append(server)
+
+    def loop(self, timeout=None):
+        """Poll for socket events no more than `self.timeout` or `timeout` seconds."""
+        timeout = 1
+        self.wait = True
+        try:
+            eta = time.time() + timeout
+            s_map = asyncore.socket_map
+
+            if hasattr(select, 'poll'):
+                poll_fun = asyncore.poll2
+            else:
+                poll_fun = asyncore.poll
+
+            while self.wait and (eta > time.time()):
+                poll_fun(eta - time.time(), s_map)
+        except asyncore.ExitNow:
+            pass
+
+    def recieved_response(self, response):
+        self.wait = False
+
+    def run_test(self):
+        for self.current_chain in self.message_chains:
+            self.recieved_chain = deproxy.MessageChain.empty()
+            self.client.clear()
+            self.client.set_request(self.current_chain.request)
+            self.loop()
+
+    @staticmethod
+    def run_tester(self, timeout):
+        start_time = time.time()
+        success_time = start_time
+        curtime = start_time
+        while curtime - start_time < timeout:
+            self.run_test()
+            curtime = time.time()
+            if True:
+                if curtime - success_time > self.max_deadtime:
+                    self.long_deadtime += 1
+                else:
+                    self.short_deadtime += 1
+                success_time = curtime
+            if curtime - start_time > timeout:
+                break
+
+        if curtime - success_time > self.max_deadtime:
+            self.long_deadtime += 1
+
+    def pre_test(self):
+        self.tempesta.config.set_defconfig(self.config)
+        self.create_servers()
+        self.configure_tempesta()
+        for server in self.servers:
+            server.start()
+
+        remote.tempesta.run_cmd("sysctl -w net.core.somaxconn=8192")
+        remote.tempesta.run_cmd("sysctl -w net.ipv4.tcp_max_orphans=1000000")
```

Comment:
I guess, you faced `too many orphan socket` in other tests, so don't we need the settings as global for all the tests?